languages/csharp/exercises/concept/resource-lifetime/.docs/hints.md

```diff
@@ -0,0 +1,14 @@
+## General
+
+[using statement][using-statement] documentation describes how and when to use the `using` keyword.
+
+## 1. Write to the database
+
+Consider the various methods on the database, and the states that the database transitions to with each one.
+
+## 2. Write to the database and return an indication to the caller of whether the write operation was successful
+
+Use of [`try/catch`][try-catch] is necessary in this case.
```

Comment:
```suggestion
- Use of [`try/catch`][try-catch] is necessary in this case.
```

Reply:
Maybe you can rewrite this slightly to not explicitly mention the `try/catch` keywords?